src/transaction/log_page_buffer.c

```diff
@@ -12603,3 +10783,9 @@ logpb_debug_check_log_page (THREAD_ENTRY * thread_p, void *log_pgptr_ptr)
   assert (err == NO_ERROR && is_log_page_corrupted == false);
 }
 #endif
+
+size_t
+logpb_get_memsize ()
+{
+  return (size_t) log_Pb.num_buffers * sizeof (LOG_PAGESIZE);
```

Comment:
it should be 
return (size_t) log_Pb.num_buffers * LOG_PAGESIZE;

see old definition of LOG_PRIOR_LSA_LIST_MAX_SIZE

Reply:
Nice catch :). fixed.